modules/standard/Random.chpl

```diff
@@ -484,6 +484,8 @@ module Random {
 
       :arg n: The position in the stream to skip to.  Must be > 0.
       :type n: `integral`
+
+      throws CompileError: When RandomStreamInterface.skipToNth is called.
```

Comment:
Also, compilerError is not a subclass of error, and is not thrown.  This method is basically here to be overridden, so I think it would be fair game to match the description rather than the contents of the body (so use `IllegalArgumentError` and describe when that could happen instead)